geometry_msgs/Pose2D goal
geometry_msgs/Pose2D start
int32 width
int32 height
int16_image map
int16_image cost_map
int64 number_of_iterations
int32 segment_length
int8 number_of_trees
bool minimum_distance_enabled
geometry_msgs/Point[] sketched_topology
